src/fidesops/schemas/privacy_request.py

```diff
@@ -22,7 +22,7 @@ class PrivacyRequestCreate(BaseSchema):
     started_processing_at: Optional[datetime]
     finished_processing_at: Optional[datetime]
     requested_at: datetime
-    identities: List[PrivacyRequestIdentity]
+    identities: PrivacyRequestIdentity
```

Comment:
I think I agree with @stevenbenjamin on this one- since this var refers to one `PrivacyRequestIdentity`, could we rename it across the codebase as singular `identity` or something similar?

Reply:
haha sure, 2 against 1!

Reply:
got it - to explain a bit more, in my head `email` and `phone number` are more like different aspects of the same identity, or person. So `identity_aspect_map` or `identity_prop_map` could also work. Naming is hard...

Reply:
for sure, I'm the worst at naming! I've changed to `identity` -